backend/src/schema/resolvers/notifications.spec.js

```diff
@@ -58,8 +59,15 @@ describe('given some notifications', () => {
         categoryIds,
         content: 'You have been mentioned in a post',
       }),
+      factory.create('Post', {
+        author,
+        id: 'p4',
+        categoryIds,
+        title: 'Bad Post',
+        content: 'I am bad content !!!',
+      }),
     ])
-    const [comment1, comment2, comment3] = await Promise.all([
+    const [comment1, comment2, comment3, comment4] = await Promise.all([
       factory.create('Comment', {
         author,
         postId: 'p3',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T18:56:52Z" title="Thursday, January 23rd 2020, 7:56:52 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -78,6 +86,12 @@ describe('given some notifications', () => {
         id: 'c3',
         content: 'Somebody else was mentioned in a comment',
       }),
+      factory.create('Comment', {
+        author,
+        postId: 'p4',
+        id: 'c4',
+        content: 'I am bad content in a bad comment to a bad post !!!',
```


I do like it more specific, but again, I could just be being pedantic.
like the reason category says it's bigoted, so I would probably use that adjective.
bad could be anything :open_mouth: 
feel free to ignore me

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-24T11:38:22Z" title="Friday, January 24th 2020, 12:38:22 pm +01:00">Jan 24, 2020</time>_

---


Done ;-)